EngineLayer/MatchedIonMassesListOnlyMatches.cs

```diff
@@ -47,18 +46,12 @@ IEnumerator IEnumerable.GetEnumerator()
             return matchedIonDictPositiveIsMatch.GetEnumerator();
         }
 
-        public override bool Equals(object obj)
```

Comment:
Not sure if this should be deleted or not; was removed as part of merge conflict.

Reply:
Please do not remove this. Restore if was deleted.